File: guides/micronaut-hibernate-reactive/java/src/test/java/example/micronaut/MicronautguideTest.java
package example.micronaut;

import io.micronaut.runtime.EmbeddedApplication;
import jakarta.inject.Inject;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class MicronautguideTest extends BaseMysqlTest {

    @Inject
    EmbeddedApplication<?> application;

    @Test
    void testItWorks() {
        Assertions.assertTrue(application.isRunning());
    }

}

File: guides/micronaut-hibernate-reactive/kotlin/src/test/kotlin/example/micronaut/MicronautguideTest.kt
package example.micronaut

import io.micronaut.runtime.EmbeddedApplication
import jakarta.inject.Inject
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class MicronautguideTest : BaseMysqlTest() {
    @Inject
    override var application: EmbeddedApplication<*>? = null
    @Test
    fun testItWorks() {
        Assertions.assertTrue(application!!.isRunning)
    }
}